src/messages/manifestjson.js

```diff
@@ -35,6 +35,16 @@ export const MANIFEST_FIELD_INVALID = {
   file: MANIFEST_JSON,
 };
 
+export const MANIFEST_BAD_PERMISSION = {
+  code: 'MANIFEST_BAD_PERMISSION',
+  legacyCode: null,
+  message: _('The permission type is unsupported.'),
+  description: _(singleLineString`Permissions must be strings; object
```

Comment:
Looking through https://developer.chrome.com/extensions/declare_permissions I don't see any reference to non-string values - is that a historical thing?